science/2022-11-23-science-blog-6/index_de.md

```diff
@@ -0,0 +1,270 @@
+---
+page-title: "Was sagt eine rote Warnung bei hohen Inzidenzen noch aus?"
+page-description: "Was sagt eine rote Warnung bei hohen Inzidenzen noch aus?"
+page-name: science-blog-6
+page-name_de: science-blog-6
+author: CWA-Team
+date: "2022-11-23"
+layout: science
+---
+
+In diesem Beitrag wollen wir untersuchen, welche Aussagekraft eine rote Warnung gegenüber einem niedrigen Risiko mit bzw. ohne Risikobegegnung hat. Hierzu betrachten wir den Positivenanteil der Testergebnisse, d.&nbsp;h. den Anteil der positiven Testergebnisse, in Abhängigkeit vom angezeigten Risiko. Die Daten hierfür stammen aus der [Datenspende der Corona-Warn-App](2021-10-15-science-blog-4/#3-privacy-preserving-analytics).
+
+In den vergangenen Monaten wurde kritisiert, dass in Zeiten einer hohen Inzidenz eine rote Warnung keine Aussagekraft mehr habe. Das Infektionsrisiko bestünde ganz unabhängig davon, ob eine rote Warnung in der CWA vorläge oder nicht. Außerdem sei die Häufigkeit der roten Warnungen so hoch, dass die einzelnen Warnungen keine Aussagekraft mehr haben. Wir wollen zeigen, dass dem nicht so ist. Insbesondere für Schnelltestergebnisse, die ja oft erst der Anlass für einen PCR-Test sind, war der Positivenanteil bei einer roten Warnung deutlich größer als bei einer grünen Warnung (niedriges Risiko mit Risikobegegnung). Darüber hinaus untersuchen wir, ob sich diese größeren Positivenanteile bei einer roten Warnung allein aus lokalen Schwankungen der Inzidenz erklären lassen.
+
+<!-- overview -->
+
+1. [Das Wichtigste in Kürze: Zahlen, Daten, Fakten](#1-das-wichtigste-in-kürze-zahlen-daten-fakten)
+2. [Positivenanteile (bundesweit)](#2-positivenanteile-bundesweit)
+    1. [RAT](#21-rat)
```

Comment:
Suggestion to not use "RAT" here, but "Antigen-Schnelltests". RAT is fine for figure captions, just like in the last science blog, but please not for the text itself/headlines.

Reply:
At least it should be spelled out somewhere near the topic of the article that RAT is an English abbreviation for Rapid Antigen Test (see https://en.wikipedia.org/wiki/COVID-19_rapid_antigen_test), in German "Antigen-Schnelltest".

Similarly PCR could be spelled out once as an abbreviation for COVID-19 Polymerase chain reaction test (https://en.wikipedia.org/wiki/Polymerase_chain_reaction), although at least this term is in common use as an abbreviation.

Reply:
I tried resolving this issue, let me know what you think.

- Changed the wording in the table

I have not added an explicit explanation. There are plenty places already where this gets mentioned in one way or another in the post.